# Description: tweak color computation looking to the unity launcher side
# UbuntuSpecific: tweak for unity, nothing upstream uses the color computation
# yet anyway so it's not an issue
Index: b/libgnome-desktop/gnome-bg.c
===================================================================
--- a/libgnome-desktop/gnome-bg.c
+++ b/libgnome-desktop/gnome-bg.c
@@ -2558,7 +2558,7 @@
 	if (gdk_pixbuf_get_has_alpha (pixbuf)) {
 		for (row = 0; row < height; row++) {
 			p = pixels + (row * row_stride);
-			for (column = 0; column < width; column++) {
+			for (column = 0; column < width / 2; column++) {
 				r = *p++;
 				g = *p++;
 				b = *p++;
@@ -2575,7 +2575,7 @@
 	} else {
 		for (row = 0; row < height; row++) {
 			p = pixels + (row * row_stride);
-			for (column = 0; column < width; column++) {
+			for (column = 0; column < width / 2; column++) {
 				r = *p++;
 				g = *p++;
 				b = *p++;
